### DISCUSSION TITLE
How to Change MongoDB Password for Meshcentral Deployed in Docker?

### DISCUSSION THREAD

**Discussion by Creogenesis** [Ylianst/MeshCentral#5674] [Q&A]
I have previously deployed **Meshcentral** on a VPS within my **Docker** environment. Over time, the VPS **was compromised** with a rootkit. After migrating to a new VPS, I would like to change the MongoDB password to ensure that the operation remains secure.

Could someone guide me on how to correctly change the MongoDB password for an active Meshcentral deployment in Docker?

**Comment by si458** [comment]
Simply open up thr config.json file that's inside the meshcentral-data folder and change the password in there

**Reply by Creogenesis** [reply]
```
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings": {
    "cert": "remote.somemeshdomain.com",
    "WANonly": true,
    "_LANonly": true,
    "_sessionKey": "MyReallySecretPassword1",
    "TLSOffload": true,
    "port": 443,
    "aliasPort": 443,
    "redirPort": 80,
    "redirAliasPort": 80,
    "AgentPong": 300,
    "TLSOffload": "XXX.XX.XXX.XXX",
    "_cookieIpCheck": false,
    "SelfUpdate": false,
    "AllowFraming": "false",
    "mongodb": "mongodb://mongodb:27017/mesh",
    "mongodbcol": "mesh",
    "WebRTC": "true"
  },
}
```
At the moment, this is the first section of my current meshcentral configuration. Do I simply need to add additional options with the new password and restart meshcentral? And is that all?

**Reply by si458** [reply]
So change the mongodb url to contain ur user/pass - https://ylianst.github.io/MeshCentral/meshcentral/#mongodb-setup
`mongodb://username:password@127.0.0.1:27017/meshcentral`
